README.md

```diff
@@ -1,2 +1,44 @@
 # Money On My Mind
 ## A Code for BTV App with sponsorship by CVOEO
+
+## Local Environment Setup
+
+### Requirements
+- [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- [node.js & npm](https://www.npmjs.com/get-npm)
+- [java](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+- [expo](expo.io)
+  - install expo on your computer using npm: `sudo npm install expo-cli --global`
+
+
+### Clone the Repo
+Get a copy of the app code using the command line: 
+
+`$ cd [desired directory]`
+
+`$ git clone https://github.com/codeforbtv/cvoeo-app.git`
+
+### Run the App
+- Run the npm instalation on the cvoeo-app project `$ npm install` 
+- Confirm expo can run the project by typing `$ expo start`
+
+#### View in Expo Mobile App
+By using the expo mobile app, you will be able to test the app in both iOS & Android.  However, you will need a smart phone that can connect to the same network as the computer running expo.
+
+- Download the Expo app on your phone
+- Run expo with `$ expo start`
+- Create an expo account at https://expo.io/signup
+- Open the expo
+
+#### View in Android Emulator
+- Download and install [genymotion](https://www.genymotion.com/desktop/)
+- TODO: DO THIS & COMPLETE DOCUMENTATION
```

Comment:
I'll get to this later, but if someone beats me to it... how does this work?  There is some order in which expo &/or genymotion need to be opened, right?